utils.cpp

```diff
@@ -62,9 +62,23 @@ char* get_runtime_path() {
 	return path;
 }
 
+static const char *data_dir = NULL;
+
+const char* get_data_dir(void) {
+	if (data_dir) {
+		return data_dir;
+	} else {
+		return get_runtime_path();
+	}
```

Comment:
When you compare the behavior of `get_runtime_path()` to `get_data_dir()`, it turns out that `data_dir` is _not_ guaranteed to end in a path separator (`/`). For me, this resulted in my deployment of this PR to put a bunch of files in `/` (the root directory) _prefixed_ with the word `data` rather than putting data into the `/data` volume.

My workaround was to change the `Dockerfile` to specify `/data/` as the data dir, but perhaps `set_data_dir()` should be enhanced to assure a terminating slash?

Reply:
Oops, thanks @jbalonso , will see if I can fix this.

Reply:
Should be fixed by my commit just now now. Please confirm?

Reply:
@jbalonso Heya Jason, you should now be able to set your data dir to either /data/ or /data, it shouldn't matter. Thanks for the bug report. Cheers, Matt.